src/backend/web/routes/feeds.js

```diff
@@ -51,4 +51,26 @@ feeds.get('/:id', async (req, res) => {
   }
 });
 
+feeds.post('/', async (req, res) => {
+  const feedData = req.body;
+
+  try {
+    const feed = await Feed.create(feedData);
+
+    // If the object we get back is empty, use 404
+    if (!feed) {
+      res.status(404).json({
+        message: `Feed not created for url ${feedData.url}`,
+      });
+    } else {
+      res.json(`${feed.url} has successfully been added.`);
```

Comment:
When you create a new record, you need to send back a 201, and give them enough info to get the url they need for the record.

```js
res.status(201).json({message: `Feed was successfully added.`, id: feed, url: `/feeds/${feed}` });
```